src/import/ImportWallet.tsx

```diff
@@ -271,18 +264,15 @@ const styles = StyleSheet.create({
     paddingVertical: 16,
     paddingHorizontal: 24,
   },
-  button: {
-    paddingVertical: 32,
-  },
   title: {
     textAlign: 'center',
-    ...fontStyles.h1,
+    paddingBottom: 24,
+    ...typeScale.labelSemiBoldLarge,
   },
   description: {
-    paddingTop: 16,
-    paddingBottom: 24,
-    textAlign: 'center',
-    ...fontStyles.regular,
+    paddingTop: 65,
```

Comment:
Per the designs, does this need to be static padding or is the description / button expected to be pinned to the bottom of the screen?

Reply:
I wasn't sure, the button and description are in a`KeyboardAwareScrollView`, so not sure if it would get weird when the keyboard is up if they are pinned to the bottom (what is the bottom then in that case?). Is there a way to specify a view within a ScrollView to stick to the bottom?

Reply:
there are other screens that handle this with keyboard aware scroll view, I think I've seen both cases where the button stays in the bottom behind the keyboard, or the button moves up when the keyboard shows up. Might be good to confirm with design